app/src/main/res/layout/activity_passenger_list.xml

```diff
@@ -6,4 +6,136 @@
+        android:layout_height = "wrap_content"
+        app:layout_constraintTop_toBottomOf="@id/search"
+        android:layout_marginTop = "35dp"
+        android:layout_marginStart = "25dp"
+        android:layout_marginEnd = "25dp"
+        app:cardCornerRadius = "15dp"
+        app:cardElevation="10dp"
+        android:id="@+id/bus"
+        >
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width = "match_parent"
+            android:layout_height = "wrap_content"
+            android:padding = "20dp"
+            >
+
+            <androidx.appcompat.widget.LinearLayoutCompat
+                android:layout_width = "match_parent"
+                android:layout_height = "wrap_content"
+                app:layout_constraintTop_toTopOf="parent"
+                android:id="@+id/time"
+                android:gravity="center_vertical"
+                app:layout_constraintStart_toStartOf="parent"
+                >
+
+                <androidx.appcompat.widget.AppCompatImageView
+                    android:id = "@+id/imageView"
+                    android:layout_width = "wrap_content"
+                    android:layout_height = "wrap_content"
+                    android:src = "@drawable/bus"
+                    />
+
+                <TextView
+                    android:id = "@+id/textView"
+                    android:layout_width = "match_parent"
+                    android:layout_height = "wrap_content"
+                    android:hint = "MP3809"
+                    android:textSize = "40sp"
+                    android:textAlignment="center"
+                    />
+
+            </androidx.appcompat.widget.LinearLayoutCompat>
+
+            <androidx.appcompat.widget.LinearLayoutCompat
+                android:layout_width = "match_parent"
+                android:layout_height = "wrap_content"
+                app:layout_constraintTop_toBottomOf="@+id/time"
+                android:layout_marginTop="10dp"
+                android:gravity="center_vertical"
+                >
+
+                <androidx.appcompat.widget.AppCompatImageView
+                    android:id = "@+id/imageView2"
+                    android:layout_width = "wrap_content"
+                    android:layout_height = "wrap_content"
+                    android:src = "@drawable/time"
+                    />
+
+                <TextView
+                    android:id = "@+id/textView2"
+                    android:layout_width = "match_parent"
+                    android:layout_height = "wrap_content"
+                    android:hint = "3:30PM"
+                    android:textSize = "40sp"
+                    android:textAlignment="center"
+                    />
+
+            </androidx.appcompat.widget.LinearLayoutCompat>
+
+
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+
+    </androidx.cardview.widget.CardView>
+
+    <androidx.recyclerview.widget.RecyclerView
```

Comment:
Add ID to the RecyclerView

Reply:
Add other 3 constraint tags also.
If you add only one for the top, on small screens it will go out of the screen from below.